### DISCUSSION TITLE
Manage which output/input each app uses

### DISCUSSION THREAD

**Discussion by RvN73** [File-New-Project/EarTrumpet#1412] [Ideas]
I'm looking for a way to manage which output each app uses disregarding which output is selected in Windows.

Example;
Current output in Windows IO manager is set to play on the loudspeaker in my monitor. I would like to be able to tell Chrome to disregard this and play on my headset.

**Comment by riverar** [comment]
Right click Chrome in EarTrumpet and left click on the arrows to select a target device. Easy!
